cpp/oneapi/dal/algo/covariance/backend/cpu/partial_compute_kernel_dense.cpp

```diff
@@ -53,7 +60,7 @@ static partial_compute_result<Task> call_daal_kernel_partial_compute(
     /// the logic of block size calculation is copied from DAAL,
     /// to be changed to passing the values from the performance model
     std::int64_t blockSize = 140;
-    if (ctx.get_enabled_cpu_extensions() == dal::detail::cpu_extension::avx512) {
+    if (ctx.get_enabled_cpu_extensions() == CPU_EXTENSION) {
```

Comment:
not sure if this is right way to handle this - the idea of this block is that this is avx512 specialized code here  and as well as in some other algorithms. It assume avx512 isa and i'm not sire that it would work that easily on ARM. We probably can have a better approach defined for architecture/ISA specific implementation in algorithms.